modules/database-commons/src/main/java/org/testcontainers/delegate/AbstractDatabaseDelegate.java

```diff
@@ -0,0 +1,61 @@
+package org.testcontainers.delegate;
+
+import java.util.Collection;
+
+/**
+ * @param <CONTAINER>  testcontainers container
+ * @param <CONNECTION> connection to the database
+ * @author Eugeny Karpov
+ */
+public abstract class AbstractDatabaseDelegate<CONTAINER, CONNECTION> implements DatabaseDelegate {
+
+    /**
+     * Testcontainers container
+     */
+    protected CONTAINER container;
```

Comment:
unused variable

Reply:
It does used in children - JdbcDatabaseDelegate and CassandraDatabaseDelegate.

Reply:
children should store it themselves

Reply:
Done.